src/otx/core/model/module/classification.py

```diff
@@ -43,7 +46,7 @@ def __init__(
         torch_compile: bool,
         optimizer: list[OptimizerCallable] | OptimizerCallable = lambda p: torch.optim.SGD(p, lr=0.01),
         scheduler: list[LRSchedulerCallable] | LRSchedulerCallable = torch.optim.lr_scheduler.ConstantLR,
-        metric: MetricCallable | None = None,
+        metric: MetricCallable = lambda: Accuracy(task="multiclass"),
```

Comment:
Is this working without adding `num_classes`?
```suggestion
        metric: MetricCallable = lambda num_classes: Accuracy(task="multiclass", num_classes=num_classes),
```

Reply:
Yes, it works well since it's not mandatory.

Reply:
BTW, I edited this -> https://github.com/openvinotoolkit/training_extensions/pull/2964/commits/5a50299ddc614f9552abc2f787acd7a2c35c9c4a